### PR TITLE
Update restrictions

### PR DESCRIPTION
**Pull request by marfcg**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Bumped the build number (if the version is unchanged)

Updated restrictions for win, py2k and py34.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,17 +11,17 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: true  # [py2k]
+  skip: true  # [py2k and py34 and win]
```
Comment (xmnlab) [inline recipe/meta.yaml]: I think maybe 'or' would be the correct operator here

Comment (marfcg) [reply recipe/meta.yaml]: You're absolutely right.
Correction made.

**Comment by marfcg** [comment]
@conda-forge-admin, please rerender

**Comment by xmnlab** [comment]
LGTM

**Pull request merged**